default.nix

```diff
@@ -191,6 +191,17 @@ let
       staging.wallet = mkDocker { environment = "mainnet-staging"; };
       testnet.wallet = mkDocker { environment = "testnet"; };
     };
+    acceptanceTests = let
+      acceptanceTest = pkgs.callPackage ./scripts/test/acceptance;
+      mkTest = environment: {
+        full  = acceptanceTest { inherit environment; resume = false; };
+        quick = acceptanceTest { inherit environment; resume = true; };
+      };
+    in {
+      mainnet = mkTest "mainnet";
+      staging = mkTest "mainnet-staging";
+      testnet = mkTest "testnet";
+    };
```

Comment:
instead of repeating the cluster names all over the place, it may help to make a config file like we have on daedalus
https://github.com/input-output-hk/daedalus/blob/57c01ecebc203945126c7e8686a09794e856db30/release.nix#L27-L30
then we can just edit 1 spot to add another cluser

Reply:
I never liked that config file in daedalus. It just contains three words `mainnet staging testnet` and these are very rarely going to change.

But I have added a library function which helps generate these attrsets from a central config.